04/style/mobile.css

```diff
@@ -0,0 +1,50 @@
+.menu__input {
+	display: none;
+}
+.menu__label {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	position: fixed;
+	top: 30px;
+	right: 30px;
+	width: 5rem;
+	padding: 1rem;
+	border: 1px solid black;
+	border-radius: 10%;
+}
+.menu {
+	display: block;
```

Comment:
Domyślnie menu ma być niewidoczne na mobile, dopiero po kliknięciu ma się pokazać ;)

Reply:
no rzeczywiście na produkcję wrzuciłem a zapomniałem to przestawić już poprawiam.